.github/workflows/default.yml

```diff
@@ -0,0 +1,83 @@
+name: PHP Workflow
```

Comment:
It will produce the following workflow page.
https://github.com/shahariaazam/QuickStart/actions/runs/4413625939